quartz_solar_forecast/data.py

```diff
@@ -163,22 +163,33 @@ def make_pv_data(site: PVSite, ts: pd.Timestamp) -> xr.Dataset:
             raise ValueError("Multiple sites found matching the given latitude and longitude.")
         else:
             site_id = matching_site_ids[0]
-            live_generation_wh = get_solaredge_data(site_id)
+            live_generation_kw = get_solaredge_data(site_id)
     elif site.inverter_type == 'enphase':
-        live_generation_wh = get_enphase_data(system_id)
+        live_generation_kw = get_enphase_data(system_id)
+
+        if live_generation_kw is not None:
+            # get the most recent data
+            recent_pv_data = live_generation_kw[live_generation_kw['timestamp'] <= ts]
+            power_kw = np.array([np.array(recent_pv_data["power_kw"].values, dtype=np.float64)])
+            timestamp = recent_pv_data['timestamp'].values
+            
+            lon = [site.longitude] * len(power_kw)
+            lat = [site.latitude] * len(power_kw)
+            pv_id = [1] * len(power_kw)
+        else:
+            # make fake pv data, this is where we could add history of a pv system
+            power_kw = [[np.nan]]
+            timestamp = [ts]
     else:
         # If no inverter type is specified, use the default value
-        live_generation_wh = np.nan
-
-    # Combine live data with fake PV data, this is where we could add history of a PV system
-    generation_wh = [[live_generation_wh]]
-    lon = [site.longitude]
-    lat = [site.latitude]
-    timestamp = [ts]
-    pv_id = [1]
+        power_kw = [[np.nan]]
+        timestamp = [ts]
+        lon = [site.longitude] * len(power_kw)
```

Comment:
When there is no inverter used in this bit does the lon and lat need to be in these dimensions? Can they not just be `[site.longitude]` and `[site.latitude]`? Same question for the pv_id. Let me know if that's not the case

I am wondering why the forecast came out as just 0 when the inverter wasn't used in the plot and feel something is a bit off about that